keras_cv/models/task.py

```diff
@@ -136,7 +136,7 @@ def from_preset(
             return model
 
         weights = keras.utils.get_file(
-            "model.h5",
+            "model.weights.h5",
```

Comment:
Why this change? This will break out `ImageClassifier` presets for sure.

Reply:
Oops, I had a fix for this but must not have pushed